bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -91,6 +118,19 @@ class Base extends Component {
     }
   }
 
+  componentWillUnmount() {
+    const fullscreenChangedEvents = [
```

Comment:
pls, declare this variable either a class property or a global variable, not duplicate it.